packages/terra-site/src/examples/alert/LongTextExample.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Alert from 'terra-alert';
+import Button from 'terra-button';
+
+class LongTextExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allTextShown: false,
+    };
+    this.handleShowMore = this.handleShowMore.bind(this);
+    this.handleShowLess = this.handleShowLess.bind(this);
+  }
+
+  handleShowMore() {
+    this.setState({ allTextShown: true });
+  }
+
+  handleShowLess() {
+    this.setState({ allTextShown: false });
+  }
+
+  render() {
+    const alertStartTagStr = '<Alert type={Alert.Types.INFO} title="Gettysburg Address:" >';
+    const alertContentStr = '<span>Four score and seven years ago our fathers brought forth on this continent, a new nation, . . .</span>';
+    const alertEndTagStr = '</Alert>';
+    const moreTextDisplayStyle = {};
+    if (this.state.allTextShown) {
+      moreTextDisplayStyle.display = 'inline';
+    } else {
+      moreTextDisplayStyle.display = 'none';
+    }
+    const showMoreButtonDisplayStyle = {};
+    if (this.state.allTextShown) {
+      showMoreButtonDisplayStyle.display = 'none';
+    } else {
+      showMoreButtonDisplayStyle.display = 'inline';
+    }
+    return (
+      <div>
+        <div dir="ltr">
+          <h3>Alert of type info with custom title and long text with show more/less</h3>
```

Comment:
The show more/less examples should probably use the toggle component: http://engineering.cerner.com/terra-core/#/site/toggle